source/docs/software/kinematics-and-odometry/differential-drive-odometry.rst

```diff
@@ -37,7 +37,49 @@ The ``DifferentialDriveOdometry`` class requires two mandatory arguments and one
 
 Updating the Robot Pose
 -----------------------
-The ``update`` method of the odometry class updates the robot position on the field. The update method takes in the gyro angle of the robot, along with a ``DifferentialDriveWheelSpeeds`` object representing the left and right wheel speeds of the robot. This ``update`` method must be called periodically, preferably in the ``periodic()`` method of a :ref:`Subsystem <docs/software/commandbased/subsystems:Subsystems>`. The ``update`` method returns the new updated pose of the robot.
+There are two ways to update the robot pose using the ``DifferentialDriveOdometry`` class. One involves using encoder positions whereas the other method involves using encoder velocities. The method that uses encoder positions or distances is mathematically more robust and is more advantageous for teams that are using lower CPR encoders.
+
+Both methods use overloads of the ``update`` method, which updates the robot position on the field. This method must be called periodically, preferably in the ``periodic()`` method of a :ref:`Subsystem <docs/software/commandbased/subsystems:Subsystems>`. The ``update`` method returns the new updated pose of the robot.
+
+Updating the Robot Pose using Encoder Positions or Distances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Pretty sure the heading is supposed to be ^^^^ not ~~~~

If I remember my own style guide correctly, it's =, -, ^, and then ~

Reply:
Fixed.